common/logisticspipes/utils/gui/SimpleGraphics.java

```diff
@@ -47,7 +47,6 @@ public static void drawHorizontalLine(int x1, int x2, int y, Color color, int th
 	 * @param y         the y-coordinate the line is on
 	 * @param color     the color, which the line will have
 	 * @param thickness the thickness, which the line will have
-	 * @see net.minecraft.client.gui.Gui#drawHorizontalLine(int, int, int, int)
```

Comment:
was the function on Minecraft side removed or just moved?

Reply:
I did not dig that deep into it

Reply:
![image](https://user-images.githubusercontent.com/12874766/123678913-32b60780-d847-11eb-9d48-d5969ead90a1.png)

Reply:
It doesn't really matter if it is inaccessible. It is a piece of information that shouldn't just be removed